universal-application-tool-0.0.1/app/views/ViewUtils.java

```diff
@@ -28,6 +28,11 @@ public Tag makeLocalJsTag(String filename) {
         .withType("text/javascript");
   }
 
+  /** Generates an CDN script tag for loading the javascript file found at that CDN URL. */
+  public Tag makeCdnJsTag(String cdnUrl) {
+    return script().withSrc(cdnUrl).withType("text/javascript");
+  }
```

Comment:
suggestion (blocking): please do not use a 3rd party CDN for retrieving assets. It complicates the app's CORS policy and for some civic entities may even be banned.

Reply:
Discussed offline, keeping this as a CDN for now. Creating a bug to follow up on this. Currently, the CDN is only being used in a dev Azure environment.

Reply:
I am going to play around some more with trying to get webjars working with this, if possible.